core/storage/base_model/gae_models.py

```diff
@@ -295,34 +300,35 @@ def _reconstitute_from_snapshot_id(self, snapshot_id):
 
     @classmethod
     def _get_snapshot_id(cls, instance_id, version_number):
-        """Gets snapshot id.
+        """Gets unique id for this instance and version.
 
         Args:
-            instance_id: int.
-            version_number: str.
+            instance_id: str.
+            version_number: int.
 
         Returns:
-            str.
+            str. Combination of instance_id, delimiter and version_number.
         """
         return '%s%s%s' % (
             instance_id, _VERSION_DELIMITER, version_number)
 
     def _trusted_commit(
             self, committer_id, commit_type, commit_message, commit_cmds):
-        """Main function for all commit types.
+        """Evaluates and executes commit. Main function for all commit types.
 
         Args:
-            committer_id: str.
-            commit_type: str.
+            committer_id: str. The user_id of the user who committed the change.
+            commit_type: str. Unique identifier of commit type. Possible values
+                    are in COMMIT_TYPE_CHOICES.
             commit_message: str.
-            commit_cmds: list of dicts. Should give sufficient information to
+            commit_cmds: list(dicts). A list of commands, describing changes
+                made in this model, should give sufficient information to
```

Comment:
At least state what keys are contained within each dict in the list, and their meanings, yep.